### PR TITLE
CORE-1889 Add Tapis v2 to v3 app migration guide

### PR DESCRIPTION
**Pull request by psarando**
Not sure if we need to add a link to this doc in the left Nav/TOC.

### CODE REVIEW

File: docs/services/api/tapis-v2-v3-migration.md [hunk 1]
```diff
@@ -0,0 +1,593 @@
...
+then the DE will treat that parameter as required and ignore the
+`notes.value.required` field.
+
+If the v3 parameter has an `inputMode` field with a `FIXED` value,
+then the DE will treat that parameter as hidden and ignore the
+`notes.value.visible` field.
+
+The `notes` object can contain any other custom fields
+(so the entire v2 parameter can be copied as the v3 `notes` field),
+which will be ignored by the DE, with the following exceptions.
```
Comment (slr71) [inline docs/services/api/tapis-v2-v3-migration.md]: I was expecting exceptions to this rule to appear immediately following this paragraph. Is this referring to the exceptions mentioned above this paragraph?

**Review by slr71 — APPROVED** [review]
Looks good to me! 👍

File: docs/services/api/tapis-v2-v3-migration.md (hunk 1 above)
Comment (psarando) [reply docs/services/api/tapis-v2-v3-migration.md]: No. I had trouble phrasing this line.

There are exceptions mentioned above this paragraph, and the `notes` field can contain any _other_ custom fields, which will be ignored by the DE, but there are still _other_ fields detailed in the following sections that will not be ignored.

I'll update this line to try and clear up the confusion.

**Comment by psarando** [comment]
Thanks for the review! 👍

**Pull request merged**